tools/python/openlr/quality.py

```diff
@@ -74,21 +74,41 @@ def lcs(l1, l2, eq=operator.eq):
     diff.extend(reversed(l2[:j]))
     return common[::-1], diff[::-1]
 
+def almost_equal(s1, s2, Δ=1e-5):
```

Comment:
Please, don't. I guess this is the time to stop using unicode.

Reply:
Done.